parquet-thrift/src/main/java/parquet/thrift/ThriftSchemaConvertVisitor.java

```diff
@@ -146,11 +153,23 @@ public void visit(ThriftType.ListType listType) {
 
   public MessageType getConvertedMessageType() {
     // the root should be a GroupType
-    if (currentType == null)
-      return new MessageType(currentName, new ArrayList<Type>());
+      final MessageType result;
+      if (currentType == null) {
+          result = new MessageType(currentName, new ArrayList<Type>());
+      } else {
+          GroupType rootType = currentType.asGroupType();
+          result = new MessageType(currentName, rootType.getFields());
+      }
+      logger.info("projected fields");
```

Comment:
maybe we can make a static method that builds the string to log, and then we can log it as a single call to logger.info.

That way we can write a unit test that tests the static method w/o having to test a mock logger or anything funky like that.